pages/home.html

```diff
@@ -2,9 +2,31 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="../css/bootstrap.css">
+    <link rel="stylesheet" href="../css/master.css">
     <title>Home Page</title>
   </head>
-  <body>
+  <body id=homeBackground>
+
+    <h1 id=homeHeading>Live well <br> in Dundee</h1>
```

Comment:
try not to use `<br>` in your html code. Just use the id to make the container smaller, or use Bootstrap's `.col-3` or a smaller column if necessary

Reply:
`.col-3` is a class so try not to put it in the id section
try this: `<h1 class="col col-3">Living well in Dundee<h1>`
feel free to change the column size to fit the proportions of the text

Reply:
used container class